add news items for new releases 7.8.7 and 8.0.1 (#289)
OSGeo/grass-website: content/news/2022_02_23_GRASS_GIS_7_8_7_released.md

---
title: "GRASS GIS 7.8.7 released"
date: 2022-02-23T13:38:28+02:00
layout: "news"
author: GRASS Development Team
---

### What's new in a nutshell

<a href="/images/news/grassgis787_gui_dark_mode.png">
  <img src="/images/news/grassgis787_gui_dark_mode.png"
   alt="GRASS GIS 7.8.7 GUI in dark mode"
   title="GRASS GIS 7.8.7 GUI in dark mode"
   width="45%" style="float:right;padding-left:10px;padding-top:20px">
</a>

As a follow-up to the previous GRASS GIS 7.8.6 we have published the
**new release GRASS GIS 7.8.7** with more than 40 fixes mostly focused

> on [g.extension](https://grass.osgeo.org/grass78/manuals/g.extension.html) 
> and Python 3.10 related errors in the GUI. Furthermore,
> changes to the *libgis* allowing to recover the last changed date in core
> modules and addons' manual pages were backported from newer versions.

Reviewer: ```suggestion
on the addon installation module [g.extension](https://grass.osgeo.org/grass78/manuals/g.extension.html) 
and Python 3.10 related errors in the graphical user interface (GUI). Furthermore,
changes to the internal *libgis* allowing to recover the last changed date in the manual pages
of core modules and addons has been backported from newer GRASS GIS versions.
```

... minor wording suggestions.

The overview of features in the 7.8 release series is available at
[new features in GRASS GIS 7.8](https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78).
See also our detailed announcement with the full list of changes and
bugs fixed at
[https://github.com/OSGeo/grass/releases/tag/7.8.7](https://github.com/OSGeo/grass/releases/tag/7.8.7).

**Binaries/Installer download:**

- winGRASS 7.8.7/standalone:

>   [32bit standalone installer](/grass78/binary/mswindows/native/x86/WinGRASS-7.8.7-1-Setup-x86.exe) \| [64bit standalone installer](/grass78/binary/mswindows/native/x86_64/WinGRASS-7.8.7-1-Setup-x86_64.exe)

Reviewer 2: ```suggestion
[64bit standalone installer](/grass78/binary/mswindows/native/x86_64/WinGRASS-7.8.7-1-Setup-x86_64.exe)
```

- winGRASS 7.8.7/OSGeo4W:

>   [32bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86-v1.exe) \| [64bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/v2/osgeo4w-setup.exe)

Reviewer 2: ```suggestion
[64bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/v2/osgeo4w-setup.exe)
````
